x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -115,6 +116,15 @@ export const determineToAndFrom = ({ ecsData }: { ecsData: Ecs }) => {
   return { to, from };
 };
 
+const getFiltersFromRule = (filters: string[]): Filter[] =>
```

Comment:
I think that I will do it differently to avoid weird issue if the JSON.parse failed
```
const getFiltersFromRule = (filters: string[]): Filter[] =>
  filters.reduce((acc, filterString) => {
    try {
      const objFilter =  JSON.parse(filterString);
      return [...acc, objFilter];
    } catch (e) {
      return acc;
    }
  }, []);
```